src/azure-cli/azure/cli/command_modules/sql/commands.py

```diff
@@ -547,6 +560,17 @@ def load_command_table(self, _):
         g.command('list', 'list_by_instance')
         g.command('delete', 'delete', confirmation=True, supports_no_wait=True)
 
+    managed_backup_short_term_retention_policies_operations = CliCommandType(
+        operations_tmpl='azure.mgmt.sql.operations#ManagedBackupShortTermRetentionPoliciesOperations.{}',
+        client_factory=get_sql_managed_backup_short_term_retention_policies_operations)
+
+    with self.command_group('sql midb',
+                            managed_backup_short_term_retention_policies_operations,
+                            client_factory=get_sql_managed_backup_short_term_retention_policies_operations) as g:
+
+        g.custom_command('update-retention', 'update_short_term_retention_mi', supports_no_wait=True)
+        g.custom_command('show-retention', 'get_short_term_retention_mi')
```

Comment:
What about `az sql midb delete-retention update/show` or `az sql midb delete-policy update/show` ?

Reply:
You mean as in renaming current cmdlets or as in adding a new ones for deleted databases. 

If we are talking about altering existing, we are open for suggestions. az sql midb retention-policy update/show could also be an option. However I would avoid delete-retention as this retention is for automated backups on database not database itself.

I wouldn't separate cmdlets for dropped and live databases since policy doesn't change once database is dropped. We might separate them in the future though.

Reply:
I mean rename currrent cmdlets. You can find the option you like. 

Actually, the reason I have such suggestion is that cli do not recommend commands with hyphen for the last part. The only scenario to use hyphen in command is that the abstracted subgroup only has single command, e.g.  `az stroage account show-connection-string`.